extractor/extractor.go

```diff
@@ -14,24 +16,26 @@ type Extractor struct {
 	// stream contents and resources for page
 	contents  string
 	resources *model.PdfPageResources
+	mediaBox  model.PdfRectangle
 
-	// fontCache is a simple LRU cache that is used to prevent redundant constructions of PdfFont's from
-	// PDF objects. NOTE: This is not a conventional glyph cache. It only caches PdfFont's.
+	// fontCache is a simple LRU cache that is used to prevent redundant constructions of PdfFonts
+	// from PDF objects. NOTE: This is not a conventional glyph cache. It only caches PdfFonts.
 	fontCache map[string]fontEntry
 
 	// text results from running extractXYText on forms within the page.
-	// TODO(peterwilliams): Cache this map accross all pages in a PDF to speed up processig.
+	// TODO(peterwilliams97): Cache this map accross all pages in a PDF to speed up processing.
```

Comment:
I hope to get to that right after I get high quality table extraction.

It will interesting to look at how PDF generators build invoices. They may use forms for repeated components